src/HealthCheck.Core/Configuration/IHealthCheckConfigurator.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace HealthCheck.Core.Configuration
+{
+    public interface IHealthCheckConfigurator
+    {
+        IHealthCheckConfigurator ConfigureCheckers(Func<IEnumerable<IChecker>> getChechers);
```

Comment:
Why return `IHealthCheckConfigurator`? Do you see a scenario in which one would chain multiple `ConfigureCheckers(...)` after one another?

Reply:
Yep. When we shall have more methods than one for configuration having the opportunity to use call-chain style will be more than convenient.

Reply:
Then let's add that feature when it's needed instead. You can't _know_ that there will ever be more configuration methods, right?

Reply:
Basically i am using it currently. I dont call 
configurator.ConfigureCheckers(..)
configurator.ConfigureCheckers(..)

i use 

configurator
            .ConfigureCheckers(..)
            .ConfigureCheckers(..)

looks much better

Reply:
Ok but why are you setting up your checkers in two phases in the first place? Why not just pass in all checkers in the first call?

Reply:
It just looks more vivid and descriptive from code base. Imagine you have 25 checkers which logically may be divided into 5 groups. I'd separate them this way in code. It works)